src/components/components/club-full/club-full.js

```diff
@@ -1,27 +1,23 @@
-/*
-This is the template for a single club view.
-*/
+import './club-full.scss';
```

Comment:
@alexdor moving import './club-full.scss' to the top of the imports causes visual regressions at develop build, since the base components are loaded after the current component and cannot be overwritten unless we change the css selectors specificity. 
At production build this problem is not visible, the files seem to be loaded in the correct order.
You can see the issue at the "Become a member of this club" cta. What do you propose?

Reply:
You can also replicate this issue on the production build if you clear the cache. 

I've reverted this change.

I would recommend changing the css selector specificity. That's because having the import order to resolve this visual regression is a bit "hacky" :p But it's your call : )

Reply:
Adding more specificity to the selectors is easy but not optimum either. I usually can control the import order of the scss files in order to overcome this issue and have nice and beautiful bem selectors :) In this setup though I have faced some issues with that and that is why I had all the scss imports after importing the react components I depended on.
In any case, since we don't have regressions any more, I will merge the pull request for now and decide later.